caps/templates/home.html

```diff
@@ -22,8 +22,8 @@ <h2 class="mb-0">{{ percent_councils_with_plan }}% of councils have a plan</h2>
                     {% include 'icons/chevron-right.html' with classes='ml-1' role='presentation' %}
                 </a>
                 <a data-show-feedback-modal href="{{ MEDIA_URL }}data/plans/plans.zip" class="d-flex align-items-center mt-3">
-                    Download ZIP archive of {{ total_plans }} plans ({{ plan_zip_size }})
-                    {% include 'icons/download.html' with classes='ml-1' role='presentation' %}
+                    Download ZIP archive of {{ total_plans }} documents ({{ plan_zip_size }})
+                    {% include 'icons/download.html' with classes='ml-1' %}
```

Comment:
Any reason you got rid of the `role` here? We have `role="presentation"` set on our icons to tell assistive devices that they don’t need to read out the contents of the SVG (which in this case, is nothing, but it doesn’t _hurt_ to have it in there all the time.)